docs/user/openstack/customization.md

```diff
@@ -150,9 +150,17 @@ platform:
+In the `install-config.yaml`, the `machinesSubnet` is the subnet where the Kubernetes endpoints of the nodes in your cluster will be published, and where the Ingress and API ports will be created. The installer creates a network and subnet for this purpose by default, however, you have the option to prevent the installer from creating those resources and overriding them with your own by setting the value of the `machinesSubnet` to the ID of an existing OpenStack subnet. To use this feature there are a few requirements:
 
-Once that is ironed out, you can pass the ID of the subnet to the install config with the `machinesSubnet` option. This subnet will be tagged as the primary subnet for the cluster. The nodes, and VIP port will be created on this subnet. This does add some complexity to the install process that needs to be accounted for. First of all, the subnet must have DHCP enabled. Also, the CIDR of the custom subnet must match the cidr of `networks.machineNetwork`. Be aware that the API and Ingress VIPs by default will try to take the .5 and .7 of your network CIDR. To avoid other services taking the ports assigned to the api and ingress VIPs, it is recommended that users set the `apiVIP` and `ingressVIP` options in the install config to addresses that are outside of the DHCP allocation pool. Lastly, note that setting `externalDNS` while setting `machinesSubnet` is invalid usage, since the installer will not modify your subnet. If you want to add a DNS to your cluster while using a custom subnet, add it to the subnet in openstack [like this](https://docs.openstack.org/neutron/rocky/admin/config-dns-res.html).
+* The `machinesSubnet` must have DHCP enabled
+* The CIDR of `machinesSubnet` must match the cidr of `networks.machineNetwork`
+* The installer user must have permission to create ports on this network, including ports with fixed IP addresses
+
+You should also be aware of the following details:
+
+* If you plan to install with floating IPs, the `machinesSubnet` must be attached to a router that is connected to the `externalNetwork`
+* The API and Ingress VIPs by default will try to take the .5 and .7 of your network CIDR. To avoid other services taking the ports assigned to the api and ingress VIPs, it is recommended that users set the `apiVIP` and `ingressVIP` options in the `install-config.yaml` to addresses that are outside of the DHCP allocation pool.
```

Comment:
```suggestion
* By default, the API and Ingress VIPs try to take the .5 and .7 of your network CIDR. To prevent other services from taking the ports that are assigned to the API and Ingress VIPs, set the `apiVIP` and `ingressVIP` options in the `install-config.yaml` to addresses that are outside of the DHCP allocation pool.
```